hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/replication/VerifyReplication.java

```diff
@@ -179,9 +198,12 @@ public void map(ImmutableBytesWritable row, final Result value, Context context)
         if (versions >= 0) {
           scan.readVersions(versions);
         }
+        int reCompareThreads = conf.getInt(NAME + ".reCompareThreads", 1);
+        reCompareExecutor = buildReCompareExecutor(reCompareThreads, context);
```

Comment:
I'm not entirely certain when `cleanup` gets called. Does it make sense to only build the reCompareExecutor if it is null? Or do we want to build a new executor on every call to `map`? 

Especially b/c the executor is a static member variable